python_passion_project/urls.py

```diff
@@ -17,7 +17,11 @@
 from django.contrib import admin
 from django.urls import path, include
 
-urlpatterns = [path("admin/", admin.site.urls), path("", include("retail_app.urls"))]
+urlpatterns = [
+    path("admin/", admin.site.urls),
+    path("", include("retail_app.urls")),
+    path("api/v1/products/", include("api.urls")),
```

Comment:
This should be `path("api", include("api.urls"))` The rest of the path would then be defined in the api.urls file above

Reply:
i was thinking something like that would be better... this would make it more flexible for version updates in the future, right?